inc/limit_login_attempts/namespace.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Altis\Security\Limit_Login_Attempts;
+
+use const Altis\ROOT_DIR;
+use function Altis\get_config;
+
+/**
+ * Bootstrap.
+ */
+function bootstrap() {
+	// Set default constants for Altis.
+	define( 'HM_LIMIT_LOGIN_DIRECT_ADDR', 'HTTP_X_FORWARDED_FOR' );
+
+	$config = get_config()['modules']['security']['limit-login-attempts'];
+
+	// Load plugin.
+	require_once ROOT_DIR . '/vendor/humanmade/hm-limit-login-attempts/hm-limit-login-attempts.php';
+
+	// If config is just `true`, enable plugin and show admin page.
+	if ( is_array( $config ) ) {
+		// Otherwise, don't display admin page and use config values.
+		add_action( 'admin_menu', __NAMESPACE__ . '\\remove_admin_page', 99 );
+
+		// Set pre_option filters on each config item.
+		foreach ( $config as $option_name => $option_value ) {
+			add_filter( 'pre_option_hm_limit_login_' . $option_name, function() {
+				return $option_value;
```

Comment:
`$option_value` isn't available in this scope, needs to passed via `use ( $option_value )`.

I'm also wondering if we should do some sanitisation / casting of specific values to bools / ints etc using a switch. It's probably ok but adds an extra level of safety for avoiding warnings / errors.

Reply:
I am now running `intval` on all options in the config except `lockout_notify` and `lockout_method` as those are the configuration settings to use strings.